test/e2e/frontend/cypress/support/commands.js

```diff
@@ -1,3 +1,5 @@
+import 'cypress-mailpit'
```

Comment:
I think I asked about this before, but just importing doesn't look to do anything here

Reply:
It only imports cy covenience methods to interact with mailpit. It does not have any value in this PR but #3999 makes full use of it